yourgov/pages/app/dashboard/index.tsx

```diff
@@ -67,37 +68,25 @@ const Page: NextPageWithLayout = () => {
 
 					<Stack gap={1.5}>
 						<H2>Popular links</H2>
-						<Columns as="ul" cols={{ xs: 1, sm: 2 }}>
-							<Card as="li" clickable shadow>
-								<CardInner>
-									<Stack gap={1}>
-										<H3>
-											<CardLink href="/not-found">
-												Industry-specific training courses
-											</CardLink>
-										</H3>
-										<Text>
-											Explore hundreds of training courses offered by TAFE and
-											other training providers.
-										</Text>
-									</Stack>
-								</CardInner>
-							</Card>
-							<Card as="li" clickable shadow>
-								<CardInner>
-									<Stack gap={1}>
-										<H3>
-											<CardLink href="/app/permits/apply-for-new-permit/mobile-food-vendor-permit">
-												Apply for a mobile food vendor permit
-											</CardLink>
-										</H3>
-										<Text>
-											Food businesses that sell food to the public need to be
-											registered before opening.
-										</Text>
-									</Stack>
-								</CardInner>
-							</Card>
+						<Columns cols={{ xs: 1, md: 4 }}>
+							<Column columnSpan={{ xs: 1, md: 3 }}>
+								<FeatureLinkList
+									links={[
+										{
+											description:
+												'Explore hundreds of training courses offered by TAFE and other training providers.',
+											href: '/not-found',
+											label: 'Industry-specific training courses',
+										},
+										{
+											description:
+												'Food businesses that sell food to the public need to be registered before opening.',
```

Comment:
I can see this already existed but I think this should just be: 

```suggestion
												'Businesses that sell food to the public need to be registered before opening.',
```